server/lib/db.ts

```diff
@@ -24,19 +25,29 @@ export class Database {
   dbName: string;
   sequelize: Sequelize;
 
-  constructor(database = env.DB_NAME) {
+  constructor(database= env.DB_NAME) {
+
     this.dbName = database;
 
-    this.sequelize = new Sequelize({
-      port: env.DB_PORT,
-      username: env.DB_USER,
-      password: env.DB_PASSWORD,
-      host: env.DB_HOST,
-      database: database,
-      dialectOptions,
-      dialect: 'postgres',
-      logging: env.DB_LOGGING,
-    });
+    if( env.isProd ){
+      this.sequelize = new Sequelize(env.DATABASE_URL,{
+        dialectOptions,
+        dialect: 'postgres',
+        logging: env.DB_LOGGING,
```

Comment:
And then these are common across both environments, so we can pull these out into a separate object and then spread it into the options here (to keep the code DRY)